src/client/.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules/
 .DS_Store
 # Excel temp files
 ~$*.xls*
+**/tmp/*
```

Comment:
I believe `tmp/` will ignore everything in any `tmp` directory anywhere in the directory tree

Reply:
this is my root folder for tmp files `src\client\__tests__\e2e\tmp` do you how exactly i should add this in .gitignore file.?i tried like this it didn't work

Reply:
`**/__tests__/e2e/tmp/*` this is working is it okay?

Reply:
It depends how specific you want to be. If you want to ignore all `tmp` dirs no matter where they are then `tmp/` will do. If you want to only ignore the `tmp` dir in that one place then `/src/client/__tests__/e2e/tmp/` should work. I'd go for `/tmp` myself :)

Reply:
fixed it ignored specific to the tests only